frontend/src/components/MarketInfo.js

```diff
@@ -44,7 +44,9 @@ const MarketInfo = ({
             ? {
                 backgroundImage: `url(${profilePic})`
               }
-            : {}
+            : {
+                backgroundImage: `url('https://user-images.githubusercontent.com/7010495/140438423-ea5354cb-d042-4af9-9ba3-fa8240210019.png')`
```

Comment:
Ezt nem innen kellene behúzni, hanem berakni a képeket a repoba.

Reply:
Ezt úgy érted, hogy a Git repónak a frontendes mappájába kerüljön be valahova ez a 2 kép, és onnan legyenek felhasználva? Vagy úgy, hogy backend részről jöjjenek?

Reply:
> Ezt nem innen kellene behúzni, hanem berakni a képeket a repoba.

Ezt úgy érted, hogy a Git repónak a frontendes mappájába kerüljön be valahova ez a 2 kép, és onnan legyenek felhasználva? Vagy úgy, hogy backend részről jöjjenek?